package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Production server for running Ember applications using FastBoot",
   "main": "./lib/server",
   "scripts": {
+    "postinstall": "scripts/is-not-legacy-vm || npm install contextify@^0.1.11",
```

Comment:
Nope. We are intentionally supporting the oldest possible node engine (which looses official support on 2016-10-01) in order to support AWS lambda. 

We do need a larger plan for dealing with node version support though.